From ab164d30bb18eb5c24ed3781ac7acde8fe61317d Mon Sep 17 00:00:00 2001
From: Alex Kozyrev <alex.kozyrev@windriver.com>
Date: Tue, 11 Dec 2018 14:14:37 -0500
Subject: [PATCH 1/1] Provide a way to set mem_stats_period_seconds in Nova.

There is no support of mem_stats_period_seconds in puppet-nova now.
We need to add a way to set it to 0 to disable QEMU memory balloon statistics.
The intention is to help with cyclictest spikes due to stats collection.

Signed-off-by: Alex Kozyrev <alex.kozyrev@windriver.com>
---
 SPECS/puppet-nova.spec | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/SPECS/puppet-nova.spec b/SPECS/puppet-nova.spec
index c40ec94..367de7b 100644
--- a/SPECS/puppet-nova.spec
+++ b/SPECS/puppet-nova.spec
@@ -19,6 +19,7 @@ Patch0007:      0007-Remove-joshuabaird-ipaclient-from-puppet-nova-requir.patch
 Patch0008:      0008-Adding-pci_weight_multiple-to-nova-scheduler-filter.patch
 Patch0009:      0009-Remove-SerialConsole-from-NovaConf.patch
 Patch0010:      0010-Remove-compute-huge.patch
+Patch0011:      0011-Provide-a-way-to-set-mem_stats_period_seconds.patch
 
 BuildArch:      noarch
 
@@ -49,6 +50,7 @@ Puppet module for OpenStack Nova
 %patch0008 -p1
 %patch0009 -p1
 %patch0010 -p1
+%patch0011 -p1
 
 find . -type f -name ".*" -exec rm {} +
 find . -size 0 -exec rm {} +
-- 
1.8.3.1

